services/app-api/handlers/reports/create.ts

```diff
@@ -83,10 +83,14 @@ export const createReport = handler(async (event, _context) => {
 
   let formTemplate, formTemplateVersion;
 
+  const isProgramPCCM =
+    unvalidatedMetadata?.programIsPCCM?.[0]?.value === "Yes";
+
   try {
     ({ formTemplate, formTemplateVersion } = await getOrCreateFormTemplate(
       reportBucket,
-      reportType
+      reportType,
+      isProgramPCCM
```

Comment:
report create now checks for PCCM radio and generates a separate template if necessary – see below